components/usage-api/usage/v1/billing.proto

```diff
@@ -35,3 +39,19 @@ message CancelSubscriptionRequest {
 
 message CancelSubscriptionResponse {
 }
+
+message GetStripeCustomerRequest {
+  oneof identifier {
+    string attribution_id = 1;
+    string stripe_customer_id = 2;
+  }
+}
+
+message GetStripeCustomerResponse {
+  StripeCustomer customer = 1;
+  string attribution_id = 2;
+}
+
+message StripeCustomer {
+  string id = 1;
+}
```

Comment:
Is it worth inlining this into the `GetStripeCustomerResponse`?

Reply:
I'm keen for this to be extended in the future more fields, so would like to keep it as a dedicated object.